announcing-pydoc-io.rst

```diff
@@ -0,0 +1,93 @@
+.. post:: Nov 4, 2016
+   :tags: pydoc, documentation, api, reference
+
+Announcing pydoc.io beta
+========================
+
+Today we're excited to announce our latest project: https://pydoc.io.
+This work was made possible by the :doc:`MOSS Grant </rtd-awarded-mozilla-open-source-support-grant>`_ from Mozilla.
+Thanks to Mozilla for giving us the money to set aside time to build this wonderful community resource.
+
+About pydoc.io
+--------------
+
+Running Read the Docs,
+we've always been proud of the documentation tooling that the Python community has.
+We prioritize prose over API documentation listings,
+and generally have a high standard of documentation in our projects.
+
+However,
+there are a specific set of use cases that API reference documentation support,
+and the Python community doesn't support them well.
+Tools like http://godoc.org and http://rubydoc.info provide these for others languages,
+and we now introduce http://pydoc.io as the Python communities version of that.
+
+What is it?
+-----------
+
+Pydoc.io will eventually auto-generate API reference documentation for every package on PyPI.
+Our beta release will support a limited set of packages that we are using for testing.
+Over time we'll expand the tooling to automatically support generating docs for any package posted on PyPI.
+
+Reference documentation is wonderful for folks who already know how a library works,
+and simply want to use it.
```

Comment:
Reference documentation is also useful for those learning the internals of a library, or want to modify a library/etc.